src/web/client/extension.ts

```diff
@@ -254,6 +258,125 @@ export function processWillSaveDocument(context: vscode.ExtensionContext) {
+                        data.userId
+                    );
+
+                    treeWebView.refresh();
+                }
+                if (data.type === "client-data") {
+                    WebExtensionContext.updateConnectedUsersInContext(
+                        data.containerId,
+                        data.fileName,
+                        data.filePath,
+                        data.userName,
+                        data.userId
+                    );
+
+                    treeWebView.refresh();
+                }
+            };
+        })
+        .catch((error) => console.error(error));
+}
+
+export function processOpenActiveTextEditor(context: vscode.ExtensionContext) {
+    try {
+        context.subscriptions.push(
+            vscode.window.onDidChangeTextEditorSelection(async () => {
+                const activeEditor = vscode.window.activeTextEditor;
+
+                const swpId = WebExtensionContext.sharedWorkSpaceMap.get(
+                    Constants.sharedWorkspaceParameters.SHAREWORKSPACE_ID
+                ) as string;
+
+                const swptenantId = WebExtensionContext.sharedWorkSpaceMap.get(
+                    Constants.sharedWorkspaceParameters.TENANT_ID
+                ) as string;
+
+                const swpAccessToken =
+                    WebExtensionContext.sharedWorkSpaceMap.get(
+                        Constants.sharedWorkspaceParameters.ACCESS_TOKEN
+                    ) as string;
+
+                const discoveryendpoint =
+                    WebExtensionContext.sharedWorkSpaceMap.get(
+                        Constants.sharedWorkspaceParameters.DISCOVERY_ENDPOINT
+                    ) as string;
+
+                if (activeEditor) {
+                    const entityId =
+                        WebExtensionContext.fileDataMap.getFileMap.get(
+                            activeEditor.document.uri.fsPath
+                        )?.entityId as string;
+                    const fileName =
+                        WebExtensionContext.fileDataMap.getFileMap.get(
+                            activeEditor.document.uri.fsPath
+                        )?.fileName;
+
+                    if (entityId) {
+                        copresenceWorker.postMessage({
+                            afrConfig: {
+                                swpId,
+                                swptenantId,
+                                discoveryendpoint,
+                                swpAccessToken,
+                            },
+                            file: {
+                                fileName: fileName,
+                                filePath: activeEditor.document.uri.fsPath,
+                            },
+                            containerId: WebExtensionContext.containerId,
+                        } as IContainerData);
+                    }
+                }
+            })
+        );
+    } catch (error) {
+        console.log(error);
```

Comment:
remove the console log from here - this should go to error telemetry function

Reply:
this done